docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Exciting MUI X improvements: v6.11.0
+description: Support for time zones, Charts in alpha, Data Grid filtering, and more.
+date: 2023-08-07T00:00:00.000Z
+authors: ['richbustos', 'josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-mid-v6-features/intro.png" alt="" width="2400" height="600" style="margin-bottom: 16px;" />
+
+Since the first v6 stable release, we've continuously rolled out new major features.
+Now that we've reached [MUI X v6.11.x](https://github.com/mui/mui-x/releases), it's time to delve into the most recent additions to MUI X v6, and why you should get the latest version now.
+
+## Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Date Pickers](#date-pickers)
+  - [Support for time zones](#support-for-time-zones)
+  - [Digital clock](#digital-clock)
+- [Data Grid](#data-grid)
+  - [Copy and paste](#copy-and-paste)
+  - [Filter on column headers](#filter-on-column-headers)
+  - [Performance enhancements: smoother scrolling and expanding rows](#performance-enhancements-smoother-scrolling-and-expanding-rows)
+- [Charts - alpha version 🎁](#charts-alpha-version)
+- [What's next](#whats-next)
+- [Feedback](#feedback)
+
+## Date Pickers
+
+### Support for time zones
+
+No more hassle displaying dates in the time zones of your choice. 🌎
+
+You can effortlessly display and select dates and times in different time zones, without worrying about conversion logic or maintaining consistency with your backend.
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop controls>
+  <source src="/static/blog/mui-x-mid-v6-features/timezone.mp4" type="video/mp4">
+</video>
+
+Set the `timezone` prop to define which time zone the Date, Time, or DateTime should be displayed in.
+
+```tsx
+<TimePicker
+  value={value}
+  onChange={setValue}
+  timezone="Pacific/Honolulu" // Can be in any timezone of your choice
+  label={'Rendered in "Pacific/Honolulu"'}
+/>
+```
+
+Check the [full instructions](https://mui.com/x/react-date-pickers/timezone/) for more information on using this feature in your application.
+
+### Digital clock
```

Comment:
It could be interesting to show the analog clock and explain why it's not great for desktop but only for mobile, it would give more weight to why this is an exciting new feature.